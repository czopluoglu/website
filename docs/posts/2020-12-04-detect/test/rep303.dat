00010001100000101111
01001001001010100001
00010010001000001110
11001101110101100110
00011101111011101001
11001101111111111011
00101000110111000100
01000001111000101101
10101111111111110111
01100101001011001111
01111101110010111011
10010100011100001010
01010011110010100110
01100101100101011111
01110000111100100110
11101111110111111110
10001011111100100011
11000110111110110011
10101101101010111100
01000001001000100111
11111001011110101110
10000000001110100011
01001010001001111110
00110111111111111111
10011110110111111101
01001101110111010011
11111101011111111110
10110111110110011111
11101000001010111001
01100011011111110110
11011001111110101111
01101111111110111111
10111100011011101000
00100000111000000110
10011011111100101111
10011111011010000111
11001111111010100010
10110001100101110011
01111101101110001111
10011111111111111111
11010100001111101011
10110001011011010110
11110001100010000111
10011101111011111110
11100111111011000110
10001000011110100100
11111101111000100100
01011001100111100011
00101001101000000011
00110111101111111111
01001101011100000001
10101001011000001110
10111110111010111111
00000001001011111111
10111011111111110111
01101011110010011011
11100101110011100111
01000101001011000111
10101001000100101010
11111100010100001001
10101101001001101011
10101111110110001010
11000110111000100010
01000001000000100100
00001000101010000100
01000011111111000011
11011101111011101011
10000001001110110100
10101011111010101100
01000010101111111111
11001000011101101111
00010101001010110110
00000100001111101010
00010001101010101000
10101111111110001110
00000010011100100111
00011011101100011111
00101100001111110101
00000011001000001111
11110111011001111101
10010100001101011010
00011111111000100101
11111011111111111010
11000010011000100111
11100000011111111011
01011101111101111100
11100111111000101100
01011011011010000011
00101000011010101111
10011100111110001011
11011101001000001111
01100000011001100001
11001001001111000100
00010000001101011111
01000001110001001111
10110111111110110110
11110110011111101110
01010001101111111011
11011011111111101110
11111101100110000111
00000001000110111111
11101001011111011110
01100100011000100111
00000001010110101011
10011101111110111111
10001011101110011100
11101011111010100111
00111101000111110110
10101001011011110111
11111111000010101010
10110111111101110111
11110011100000101110
11101101011001101111
11101101111100001010
11000101110010111011
00000100011111111110
00011110101110001010
00000100001111101101
10000001011111011010
11011100100101111110
10111101011000101000
11110111111110101111
01101001101100010000
10010110001101101111
00011001011101101010
11011111111111111111
01101111111110110010
10110111111111111011
01111101001000000010
00011000000010100111
10111000010100001111
10001001011110101001
11111000110100001011
01010101111100110110
10110110111111110111
11101111001111001111
11101110101101101011
00000101101111011011
01000111111010111111
11100111111111000111
11111101110000111011
01011001001101111110
10000100101010100000
01100001101000000100
11111001000011001010
11100000000010100001
11000001100001000001
01110101111001110110
00100111001101111110
01000010010001001111
01001000101010000000
11001101001000011001
10100000011110011110
11110111111010000011
10011000100000000011
00100000110000000110
00000000001110001010
11011001011110110111
11010001101110110011
10100000001011101011
10110111111111011111
11101111101101111111
10110110100010101111
10101001001011111111
01000000000001111100
00101111001111111111
00011101101100111101
11010001111010000011
11001100110010000000
01100001011100001011
10101000100010000111
01100011001010000011
11110001011100100110
11010010000010110110
00000001011110001010
11100111110011000100
10110100000011101000
01101001001100100011
01001110111101100111
10110111111110101010
10100111011010000111
10110010111100101010
10011101010100100111
11111001111111111111
01101100101011110111
10000101011100110011
10110000101011011111
11010001101010111100
01001001000110001100
01000001000011011111
01000001010000000110
11001111111100101101
00010001001000100010
10011111001111100111
11000100001010110101
11100011011101001011
11101010101001000000
11001100001100001110
11110101111100100111
00001111000111111111
10001000101001000111
01000100101011001011
00010011110001100010
00100100100110101111
11110011010111110111
10010111010011100110
10001101100111101111
01000000101010110010
11100000111011000010
11110101011011110110
10010000111110000011
00110001010111011110
01111101100110111110
10111101011010101111
00100011001100100110
01011001010100010101
00010000001001001111
00101000011010100111
01001001000011000010
00001001000100011101
11101110011110101111
10110001100010000100
10101011000010111110
01011001111111100111
00110011101111101000
10101111111010000110
01000001001011101111
10101000000111100111
10101111110010000110
00011001100001110111
10100111011010111110
10101101011010001010
01101101100110110011
00010000111000000001
00100011101000000010
10111111111100101110
11000000010100010010
01000101101110001101
00010101111111100111
10010101110001101001
00010111100101100100
11100011111010111110
10000000111000000011
01111010001100010011
00011100111111000111
00111000101110101001
10111111110001110110
01000111100001000011
01001000101010011110
00101000100000011111
