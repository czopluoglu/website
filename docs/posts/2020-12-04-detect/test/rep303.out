-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep303.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.36
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.6713
                   IDN index value:   0.6789
                           Ratio r:   0.5739

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    1    3    2    3    1    3    3    1
    1    4    4    4    4    4    4    4    1    2

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    3    7   10   11   19

   -----------CLUSTER  2-------------

    2    5   20

   -----------CLUSTER  3-------------

    4    6    8    9

   -----------CLUSTER  4-------------

   12   13   14   15   16   17   18
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+++---+-++---+-----
   +d++-+++--+---+----+
   ++d+++----+++---+--+
   +++d+++--++++-------
   --++d-----+--+--+-+-
   -+++-d--++---+++-+-+
   -+-+--d++--++--+----
   ++----+d+-++-+----+-
   -----+++d---++-+----
   +--+-+---d++++-++---
   +++++--+-+d++-+++-+-
   --++--++-++d+----+-+
   --++--+-++++d-+-----
   ----++-+++---d-++++-
   ++---+----+-+-d+-+--
   -----++-+++--++d++++
   --+-+----++--+-+d+-+
   -----+-----+-++++d++
   ----+--+--+--+-+-+d-
   -++--+-----+---+++-d
 
 No cross validation for this DETECT run
